plugins/Makefile.mk

```diff
@@ -23,3 +23,8 @@ include plugins/dmx4linux/Makefile.mk
 include plugins/e131/Makefile.mk
 include plugins/uartdmx/Makefile.mk
 endif
+
+dist_noinst_SCRIPTS += plugins/convert_README_to_header.sh
+
+plugins/%/PluginDescription.h: plugins/%/README.md plugins/convert_README_to_header.sh plugins/Makefile.mk
```

Comment:
Yeah, I saw that too, but since I'm not very familiar with automake (just a bit with pure make) I couldn't find a good solution. I don't want to put these lines in every plugin because then e.g. a filename change of the shell script or dependency changes would require all the plugin Makefiles to be updated.